pom.xml

```diff
@@ -12,28 +12,30 @@
 	<url>https://github.com/deviceinsight/helm-maven-plugin</url>
 
 	<properties>
+		<!-- Compiler -->
 		<kotlin.version>1.8.10</kotlin.version>
 		<kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+		<!-- Maven -->
 		<maven.version>3.6.2</maven.version>
```

Comment:
Maybe we should update to 3.9.1? (3.9 includes the update to JDK 8 as a baseline)

Reply:
Given that the warning also appears on use of the current plugin with the new Maven, I think we'll have to address it at some point.
But OK, it doesn't have to be in this PR.